src/command_modules/azure-cli-servicefabric/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.1.13
+++++++
+* Fix for issue 7910.
```

Comment:
Please remove the version number (we figure all that out on release and it is why the CI is failing). The message should be descriptive like:
`* sf cluster list: Fix issue 'ClusterListResult is not iterable'.`